src/helpers/UserMessageException.ts

```diff
@@ -0,0 +1,6 @@
+export class UserMessageError extends Error {
```

Comment:
Can we rename this file to `UserMessageError`?

Reply:
mb, missed that on a refactor.

Reply:
done